src/components/forms/checkbox.scss

```diff
@@ -0,0 +1,43 @@
+@import "../../colors";
+@import "../../frameless";
+
+.row {
```

Comment:
If we're wrapping everything in a wrapper `.checkbox-row` div, we may as well use it to wrap the styles here